llvm/lib/Target/AMDGPU/AMDGPUAsanInstrumentation.cpp

```diff
@@ -179,6 +181,46 @@ void instrumentAddress(Module &M, IRBuilder<> &IRB, Instruction *OrigIns,
   return;
 }
 
+void instrumentAddress(Module &M, IRBuilder<> &IRB, Instruction *OrigIns,
+                       Instruction *InsertBefore, Value *Addr,
+                       MaybeAlign Alignment, TypeSize TypeStoreSize,
```

Comment:
There's no such thing as no alignment, do not use MaybeAlign (we really should just get rid of it)